llvm/lib/Target/AIE/AIE2InstrInfo.h

```diff
@@ -179,6 +179,12 @@ class AIE2InstrInfo : public AIE2GenInstrInfo {
   std::optional<const VExtractOpInfo>
   getVExtractOpInfo(const MachineInstr &MI) const override;
 
+  unsigned getMaxLoadStoreSize() const override;
+
+  bool canCombineWithLoadStore(const MachineInstr &MI) const override;
+
+  bool isSplittableType(const LLT Ty) const override;
```

Comment:
Nit: Maybe explain what `isSplittable` means here? I guess we mean it is splittable for memory operations